_layouts/default.html

```diff
@@ -4,7 +4,18 @@
     <meta charset="utf-8">
     <title>{% if page.title %}{{ page.title }}{% if page.subtitle %} {{ page.subtitle }}{% endif %} | {% endif %}{{ site.title }}</title>
```

Comment:
I think we need to revisit this with my switchup? Maybe strip/format the subtitle better